bazarr.py

```diff
@@ -150,23 +128,30 @@ def daemon(bazarr_runner=lambda: start_bazarr()):
             else:
                 bazarr_runner()
 
+    def should_stop():
+        return daemonStatus.should_stop()
 
-    bazarr_runner = lambda: start_bazarr()
-
-    should_stop = lambda: False
+    def bazarr_runner():
+        start_bazarr(daemonStatus)
 
     daemonStatus = DaemonStatus()
 
+    def force_shutdown():
+        # force the killing of children processes
+        daemonStatus.force_stop()
+        # if a new SIGTERM signal is caught the standard behaviour should be followed
+        signal.signal(signal.SIGTERM, signal.SIG_DFL)
```

Comment:
So here we're telling it's the end of the game to children processes and updating the behaviour of the SIGTERM to be the default one.

And then "interrupting" ourselves.

To be honest I didn't have the opportunity to test this bit... Since it would have implied that the bazarr children processes wouldn't exit !